docs/examples/driver_examples/Qcodes example with Alazar ATS9360.ipynb

```diff
@@ -997,6 +997,20 @@
    "source": [
     "plot_by_id(20)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Looking for extra functionalities?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
I would like to write something like.

It is expected that the user will implement acquisition to fit their needs one can visit this link for some examples of other controllers ....